blog/releases/announcing-knative-v0-23-release.md

```diff
@@ -0,0 +1,202 @@
+
+- Change the default post-install job to use sslip.io rather than xip.io. [#11298](https://github.com/knative/serving/pull/11298)
+
+#### 💫 New Features & Changes
+
+- The stats scraping in the autoscaler is now sensitive to the EnableMeshPodAddressability setting. A restart of the autoscaler is required for the setting to take effect if changed. [#11161](https://github.com/knative/serving/pull/11161)
+- The state keeping in the activator is now sensitive to the EnableMeshPodAddressability setting. A restart of the activator is required for the setting to take effect if changed. [#11172](https://github.com/knative/serving/pull/11172)
+- Tightens the heuristic for mesh being enabled in the service scraper. We now expect all errors to be related to mesh (i.e. 503 status code). This prevents accidentally falling in to service scrape mode when errors are encountered for other reasons. [#11174](https://github.com/knative/serving/pull/11174)
+
+#### 🐞 Bug Fixes
+
+- Added schemas to all CRDs. [#11244](https://github.com/knative/serving/pull/11244)
+- Changed the rollout behavior of application deployment changes (due to Knative upgrade for example) to never have less ready posd than required. [#11140](https://github.com/knative/serving/pull/11140)
+- Rate limits digest resolution (10 QPS, retry back-off 1s to 1000s) to prevent exceeding quota at remote registries [#11279](https://github.com/knative/serving/pull/11279)
+- Revision replicas shut down 15s quicker. [#11249](https://github.com/knative/serving/pull/11249)
+- The activator's proxy is now sensitive to the EnableMeshPodAddressability setting. [#11162](https://github.com/knative/serving/pull/11162)
+- Update the User-Agent used during tag resolution [#10590](https://github.com/knative/serving/pull/10590)
+
+
+### Eventing v0.23
+
+<!-- Original notes are here: https://github.com/knative/eventing/releases/tag/v0.23.1 -->
+
+#### 🚨 Breaking or Notable Changes
+
+- Remove {`eventing`,`flows`,`messaging`} `v1beta1` APIs [#5201](https://github.com/knative/eventing/pull/5201)
+- Promote PingSource to v1 [#5324](https://github.com/knative/eventing/pull/5324)
+- Remove `APIServerSource`, `ContainerSource`, `PingSource` and `SinkBinding` v1alpha2 APIs [#5318](https://github.com/knative/eventing/pull/5318)
+- Remove `APIServerSource`, `ContainerSource`, `PingSource` and `SinkBinding` v1beta1 APIs [#5319](https://github.com/knative/eventing/pull/5319)
+- Remove `APIServerSource` and `SinkBinding` v1alpha1 APIs [#5317](https://github.com/knative/eventing/pull/5317)
+
+#### 💫 New Features & Changes
+
+- `InMemoryChannel` can be used independent of Eventing Core. Install by using `in-memory-channel.yaml` no need to install anything else to be able to use the feature. [#5355](https://github.com/knative/eventing/pull/5355)
+- `MTChannelBasedBrokers` will see improved filter performance for triggers with few or no trigger filters [#5288](https://github.com/knative/eventing/pull/5355)
```

Comment:
```suggestion
- Multi-tenant, channel-based brokers now have improved filter performance for Triggers with few or no filters. [#5288](https://github.com/knative/eventing/pull/5355)
```

How many is "few", can we specify? e.g. "up to X filters"?

Reply:
Don't know, but @n3wscott  might know